docs/why_mf_next.md

```diff
@@ -74,9 +75,11 @@ But the stability requirements prevent any major cleanup.
   grammatical agreement requires words outside select / plural to change.
   See https://en.wikipedia.org/wiki/Agreement_(linguistics)
 - Patterns in the date / time / number placeholders are bad i18n, should use skeletons
-- No official support for gender. It can be done with `select`, but it
-  is not the same thing (same as the difference between an `enum` and integer/strings). Developers can use masculine/feminine, masc/fem, male/female, etc.
-- Formatting for “parameters” known at compile time
+- No official support for gender. It can be done with `select`, but that has problems for localization tooling:
+  strings can't account for the difference in gender categories across languages, and there are no standard “enum” identifiers:
+  developers use masculine/feminine, masc/fem, male/female, etc.
```

Comment:
This has: double colons: and needs more work. Also, we don't support gender in 2.0-45 either.

```suggestion
- Unable to support gender selection. 
  Using existing formatters such as `select` doesn't account
  for the grammatical needs of different gender categories across languages.
  Tools have no way to know what modifications are needed
  and developers have to understand the needs of current and future languages to succeed.
```

Reply:
> Unable to support gender selection. 
That is misleading, because people do use select for gender support. It just supports is badly!
Suggest:

Gender selection is badly supported.